app/javascript/lib/metrics_table_sort.js

```diff
@@ -3,9 +3,19 @@ import { initSortableColumn } from './table_sort';
 // Sum site-level counts into org-level values for org row.
 function setupOrgLevelCounts() {
     $(".org-metrics").each(function() {
+        let denominator = 0;
         let attention_count = 0;
         let communication_count = 0;
         let interaction_count = 0;
+
+        if ($(this).find('tr.site').length == 0) {
```

Comment:
If there is ONLY an org and no sites, we need to use the data-js-count value from the org. Otherwise, count it from the sites.

Reply:
ah, makes sense